include/deal.II/base/function_lib.h

```diff
@@ -965,6 +1016,92 @@ namespace Functions
      * in all components.
      */
     const unsigned int selected;
+
+    /**
+     * Flag that controls wether we rescale the value when the radius changes.
+     */
+    bool integrate_to_one;
```

Comment:
You don't have a way to set this other than in the constructor, so it should be `const`.